packages/babel-plugin-transform-classes/test/fixtures/loose/program-with-use-strict-directive/input.js

```diff
@@ -0,0 +1,7 @@
+"use strict";
+
+class Foo {
+    method() {
+        
+    }
+}
```

Comment:
Test showing the case where the directive is already preset.+1: 

Could you also please add a test where the directive isn't in the program but in a parent node? like that:

```js
function t() {
  "use strict";
  class Foo {
    method() {}
  }
}
```